packages/video_player/video_player_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.3
```

Comment:
Since it is a non-breaking change to the public API, we would need to update the minor version so 2.1.0
https://dart.dev/tools/pub/versioning#semantic-versions